spec/acceptance/management_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper_acceptance'
+
+describe 'rancher::management' do
+  context 'puppet apply' do
+    it 'is expected to apply and be idempotent' do
+      pp = <<-EOS
+class { 'rancher::management':
+  rancher_manager_port => '8080',
+}
+EOS
+
+      apply_manifest(pp, :catch_failures => true)
+      apply_manifest(pp, :catch_changes => true)
+    end
+
+    context 'is expected to be running on the default port' do
+      before :each do
+        sleep 5 # Give it time to start.
```

Comment:
Unfortunately, due to the nature of how beaker-rspec runs, this won't delay the check. This sleep will just happen when the Ruby spec runs:

```
centos-72-x64 10:38:25$ /bin/sh -c curl\ 0.0.0.0:8080
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  curl: (56) Recv failure: Connection reset by peer
```

You have to put a sleep exec into the host itself to work:

```ruby
describe command('sleep 5 && echo "Give Rancher time to start"') do
  its(:exit_status) { should eq 0 }
end